qir/qat/Apps/Qat/Qat.cpp

```diff
@@ -235,6 +238,15 @@ int main(int argc, char** argv)
                 }
             });
 
+        generator->registerProfileComponent<GroupingPassConfiguration>(
+            "grouping", [](GroupingPassConfiguration const& cfg, ProfileGenerator* ptr, Profile& profile) {
+                auto& mam = profile.moduleAnalysisManager();
+                mam.registerPass([&] { return GroupingAnalysisPass(cfg); });
+                auto& ret = ptr->modulePassManager();
+                // ret.addPass(GroupingAnalysisPassPrinter());
```

Comment:
nitpick: It would be good to include a comment as to why this line is left here, so folks know they can uncomment it if they want some additional printed output.